lib/shared/nav_bar.dart

```diff
@@ -58,31 +59,38 @@ class _ScreenWithNavBarState extends State<ScreenWithNavBar>
         label: label);
   }
 
+  AppBarLeft _getAppBarLeft() {
+    if (_currentTab == Tab.home) return AppBarLeft(iconType: LeftIconType.LOGO);
+    if (_currentTab == Tab.feed) return AppBarLeft(label: '산책피드');
+    return AppBarLeft(label: '산책일지');
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(),
+      appBar: SharedAppBar(
+        leftOptions: _getAppBarLeft(),
+        rightOptions: AppBarRight(icons: [
+          RightIcon(name: 'calendar', onTap: () => {}),
+          RightIcon(name: 'menu', onTap: () => {Get.toNamed('/settings')})
+        ]),
+      ),
       body: _pages[_currentTab],
       floatingActionButton:
           _currentTab == Tab.feed ? const WriteFloatingButton() : null,
       bottomNavigationBar: Container(
         padding: const EdgeInsets.only(top: 7),
         decoration: BoxDecoration(
-            border:
-                Border(top: BorderSide(color: ThemeColor.gray2, width: 1))),
+            border: Border(top: BorderSide(color: ThemeColor.gray2, width: 1))),
         child: BottomNavigationBar(
           currentIndex: _allowedRoutes.indexOf(_currentTab),
           onTap: (index) => _changeTab(index),
           selectedItemColor: ThemeColor.black,
           selectedLabelStyle: const TextStyle(
-              fontWeight: FontWeight.w600,
-              fontSize: 10,
-            letterSpacing: -1),
+              fontWeight: FontWeight.w600, fontSize: 10, letterSpacing: -1),
           unselectedItemColor: ThemeColor.gray4,
           unselectedLabelStyle: const TextStyle(
-              fontWeight: FontWeight.w600,
-              fontSize: 10,
-              letterSpacing: -1),
+              fontWeight: FontWeight.w600, fontSize: 10, letterSpacing: -1),
```

Comment:
아 얘들은 폰트 적용이 안돼있었네.. 추가되면 제가 넣어둘게용 ~~
```suggestion
              fontWeight: FontWeight.w600, fontSize: 10, letterSpacing: -0.01 * 10),
```
이렇게 해야 letterSpacing 적용되는 것이었따 .. 퍼센트를 우리가 무시해버림

Reply:
@narinn-star 
-0.1이어야 한다는건가? 그게 무슨뜻이야?ㅜ

Reply:
오 아닝 피그마에 -1%라고 돼있는데 이거 백분율이니께 0.01이잖아? 그래서 우리 폰트크기랑 -0.01만큼 곱해줘야해 !!

Reply:
@narinn-star 오? 그러네 %였구나.. 오키 나중에 일괄루 수정해야겠네

Reply:
다 수정해둬쓰 !